packages/ethereum/sdk/src/config/alephzero-testnet.ts

```diff
@@ -0,0 +1,50 @@
+import { toAddress, ZERO_ADDRESS } from "@rarible/types"
+import { Blockchain } from "@rarible/api-client"
+import { id32 } from "../common/id"
+import type { EthereumConfig } from "./type"
+import { FEE_CONFIG_URL } from "./common"
+
+export const alephzeroTestnetConfig: EthereumConfig = {
```

Comment:
Please, use extendConfig to avoid adding zero addresses (ex. packages/ethereum/sdk/src/config/palm-testnet.ts)